vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -259,3 +282,16 @@ func BuildAndRegisterOpenAPIVersionedService(servePath string, webServices []*re
 	}
 	return o, o.RegisterOpenAPIVersionedService(servePath, handler)
 }
+
+func (c *cache) Get() ([]byte, string, error) {
```

Comment:
Make sure you move this function next to the "cache" object definition so that it's easier to read/find.

Reply:
Gotcha, fixed.